tokman/app.py

```diff
@@ -0,0 +1,115 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import os
+
+from datetime import datetime
+from pathlib import Path
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from github import GithubIntegration
+
+try:
+    from flask_restx import Api, Resource
+except ModuleNotFoundError:
+    from flask_restplus import Api, Resource
+
+
+api = Api()
+db = SQLAlchemy()
+github_integration = None
+token_renew_at = None
+
+
+def create_app():
+    app = Flask(__name__)
+    app.config.from_envvar("TOKMAN_CONFIG")
+
+    log_level = os.getenv("LOG_LEVEL", "info")
+    log_level = getattr(logging, log_level.upper())
+    logging.basicConfig(level=log_level)
+
+    private_key = Path(app.config["GITHUB_APP_PRIVATE_KEY"]).read_text()
+    app_id = int(app.config["GITHUB_APP_ID"])
+    global github_integration
+    github_integration = GithubIntegration(app_id, private_key)
+    global token_renew_at
```

Comment:
Can we get rid of `global` variables? What about having all that code in one class?

Reply:
Moved them in the application context.